examples/src/main/java/nl/knaw/dans/lib/dataverse/ExampleBase.java

```diff
@@ -28,15 +29,15 @@
 public abstract class ExampleBase {
 
     protected static DataverseClient client;
-    protected static ObjectMapper mapper = new ObjectMapper();
+    protected static ObjectMapper mapper = new ObjectMapper().configure(com.fasterxml.jackson.databind.SerializationFeature.WRITE_NULL_MAP_VALUES, false);
```

Comment:
Maybe we should not put fields with null values in a json string. See https://github.com/IQSS/dataverse/issues/11986
This attempt does not omit null values